sos/plugins/nvme.py

```diff
@@ -0,0 +1,119 @@
+### This program is free software; you can redistribute it and/or modify
+## it under the terms of the GNU General Public License as published by
+## the Free Software Foundation; either version 2 of the License, or
+## (at your option) any later version.
+
+## This program is distributed in the hope that it will be useful,
+## but WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+## GNU General Public License for more details.
+
+## You should have received a copy of the GNU General Public License
+## along with this program; if not, write to the Free Software
+## Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+from sos.plugins import Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin
+import os, subprocess
+
+class NvmePlugin(Plugin, RedHatPlugin, DebianPlugin, UbuntuPlugin):
+    """Collect config and system information about NVMe devices"""
+
+    plugin_name = "nvme"
+    devices = [] 
```

Comment:
You have no package or path checks to enable this plugin, meaning it will run by default on any system. This is not likely what is wanted: either check for a well-known path, or set of paths (checking is an `any()` match), or for a specific package name.